monstache.go

```diff
@@ -82,7 +82,7 @@ var chunksRegex = regexp.MustCompile("\\.chunks$")
 var systemsRegex = regexp.MustCompile("system\\..+$")
 var exitStatus = 0
 
-const version = "6.7.10.3"
+const version = "6.7.17"
```

Comment:
Why have we bumped version beyond `6.7.10.x`?

Reply:
There have been several vulnerabilities reported for Go libraries that are picked up in our internal vuln scans. While these do not pose an attack vector for our use case. We currently use Monstache 6.7.10 while the latest version is 6.7.17.

Reply:
Right, but you're not bringing in any changes from the latest version 6.7.17 here. You're just bumping the Go version.

Reply:
we have bumped the Go version from 1.21.3 to 1.21.6 for CVE's and We upgraded the monstache version to 6.7.17 without bringing any new feature.

Reply:
Won't it be confusing, bumping this version of monstache to `6.7.17`, when it actually has the monstache code base equivalent of `6.7.10`? Also how will one distinguish this `6.7.17` from the official (but very different) `6.7.17`?

Reply:
Yes, it's confusing bumping the version without actual code changes. Shall we document this clearly in the code base that this version is based on 6.7.10 code. Or welcome your suggestion for the other versioning approach.

Reply:
I think the best is to carry on with the versioning approach that we had - keep the full monstache version & bump the extra number at the end, treating it like a build number :)

Reply:
@mix4242 .. Thanks for your inputs . What version you are suggesting here ? 6.7.17.X?

Reply:
Keep the version we had, just bump the last number :)